src/eventProcessor/GoogleAnalyticsEventProcessor.js

```diff
@@ -115,6 +115,27 @@ class GoogleAnalyticsEventProcessor {
     return 'googleAnalytics';
   }
 
+  /**
+   * Builds the POST request URL using `measurement_id` and `api_secret`
+   * query parameters if available
+   * @return {string}
+   * @private
```

Comment:
Style guide nit: private methods should end with _.
https://google.github.io/styleguide/jsguide.html#naming-method-names